ADALiOS/ADALiOS/ADAuthenticationRequest+AcquireAssertion.m

```diff
@@ -147,41 +145,44 @@ - (void)attemptToUseCacheItem:(ADTokenCacheStoreItem*)item
              return;
          }
          
+         // TODO: MRRT
```

Comment:
When I commented out the code it was a reminder to figure out if that MRRT logic was needed. I dealt with it elsewhere. I'll remove the todo.